### ISSUE TITLE
Feature Request: Ability to configure fonts in the settings

### ISSUE THREAD

**Issue by dhirschfeld** [jupyterlab/jupyterlab#4648]
This has cropped up on a number of occasions but there isn't really an issue which simply requests the ability to configure the fonts in the jupyterlab settings.

#4013 can be closed as implementing this feature will provide the requested simple way of changing the font characteristics.

#3899 and #4084 would both prefer Consolas on Windows. Since the decision in #4084 is to not provide platform default fonts both of these can be closed in favour of this feature request which would enable the user to change this for themselves.

#4608 can be closed as the question has been [answered](https://github.com/jupyterlab/jupyterlab/issues/4608#issuecomment-390431897) and this feature was probably want the user was actually after

**Comment by dhirschfeld** [comment]
To collect all relevant information here I'll repost @ian-r-rose's [comment](https://github.com/jupyterlab/jupyterlab/issues/4608#issuecomment-390686374):
> @tonyfast and @bollwyvl have a neat new extension that loads custom fonts onto the page: [`deathbeds/jupyterlab-fonts`](https://github.com/deathbeds/jupyterlab-fonts)

**Comment by dhirschfeld** [comment]
I hope the issue triage is helpful and I haven't stepped on anyone's toes!

I was looking for an issue where I could add my :+1: and found it scattered about several with none explicitly requesting the ability to configure fonts in the settings.

**Comment by sobayed** [comment]
I agree that Source Code Pro (Jupyter Lab) is not as good as Consolas (Jupyter Notebook) on Windows in terms of readability. Therefore, I would really appreciate to have a config option as this is one of the few things that is still keeping me from fully switching to Jupyter Lab.
